src/cpp/dds/domain/DomainParticipant.cpp

```diff
@@ -133,17 +133,17 @@ DomainParticipant::~DomainParticipant()
 //    return *this;
 //}
 
-//dds::sub::qos::SubscriberQos DomainParticipant::default_subscriber_qos() const
-//{
-//    return this->delegate()->get_default_subscriber_qos();
-//}
+dds::sub::qos::SubscriberQos DomainParticipant::default_subscriber_qos() const
+{
+    return this->delegate()->get_default_subscriber_qos();
+}
 
-//DomainParticipant& DomainParticipant::default_subscriber_qos(
-//        const ::dds::sub::qos::SubscriberQos& qos)
-//{
-//    this->delegate()->set_default_subscriber_qos(qos);
-//    return *this;
-//}
+DomainParticipant& DomainParticipant::default_subscriber_qos(
+        const ::dds::sub::qos::SubscriberQos& qos)
+{
+    this->delegate()->set_default_subscriber_qos(qos);
+    return *this;
+}
```

Comment:
These changes are not necessary here, they have to go in #1121

Reply:
so true...Changed